src/Symfony/Component/DependencyInjection/Definition.php

```diff
@@ -860,4 +862,34 @@ public function hasAutowiringType($type)
 
         return isset($this->autowiringTypes[$type]);
     }
+
+    /**
+     * Gets bindings.
+     *
+     * @return array
+     */
+    public function getBindings()
+    {
+        return $this->bindings;
+    }
+
+    /**
+     * Sets bindings.
+     *
```

Comment:
Maybe add some explanations, eg:
Bindings map $named arguments or type-hints to values that should be injected in the matching parameters of the constructor and of setters.